etc/kayobe/kolla.yml

```diff
@@ -331,14 +332,8 @@ kolla_build_customizations: "{{ kolla_build_customizations_common | combine(koll
 
 # Dict mapping Kolla Dockerfile ARG names to their values.
 kolla_build_args:
-  blackbox_exporter_version: "0.19.0"
-  haproxy_exporter_version: "0.13.0"
-  memcached_exporter_version: "0.9.0"
-  mysqld_exporter_version: "0.13.0"
-  node_exporter_version: "1.5.0"
+  node_exporter_version: "1.5.0" # kolla has 1.4.0
```

Comment:
Pierre updated it in yoga 6 months ago to fix a crash issue

Reply:
oops, I didn't mean to commit the comment, that was a note to myself.

Reply:
Although actually it's kind of useful